python_transport/wirepas_gateway/utils/argument_tools.py

```diff
@@ -402,6 +402,17 @@ def add_buffering_settings(self):
             ),
         )
 
+        self.buffering.add_argument(
+            "--buffering_monitor_period",
+            default=os.environ.get("WM_GW_BUFFERING_MONITOR_PERIOD", 0),
```

Comment:
Should we enable the feature by default ?
I would say that a default set to one message every 6H could be helpful for debugging.